src/ioc/index.ts

```diff
@@ -1,12 +1,28 @@
 import IoCContainer from 'ioc-lite';
+import {Users} from "../services/users";
+import {Logger} from "../services/logger";
+import {HTTP} from "../services/http";
+import {ApiConfig} from "../types";
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+type IoCResources = {
+  config: ApiConfig,
+  logger: typeof Logger,
+  http: typeof HTTP,
+  users: typeof Users
+}
 
+const config: ApiConfig = {
+  "path": "/api",
+  "resources": {
+    "users": "/users"
+  }
+}
```

Comment:
we shouldn't hardcode config, it comes from backend and can be changed. 
You can register the config in the index.ts file inside the app function.